src/pljson_parser.impl.sql

```diff
@@ -85,43 +85,27 @@ create or replace package body pljson_parser as
 
     /* use of length, so works correctly for 4-byte unicode characters (issue #169) */
     /* lengthc does not work (issue #190) */
-    if (indx > length(s.src) + s.offset_chars) then
+    if (indx > s.src_len + s.offset_chars) then
```

Comment:
It seems to me you are ignoring the warnings about handling 4-byte unicode properly.

Reply:
No. I'm not ignoring that. In my opinion it's not a problem anymore - substr_amount < 1000

Reply:
Again, opinions do not count. This is a technical matter. Please show that this change is not breaking 4-byte unicode parsing.

Reply:
> Again, opinions do not count. This is a technical matter. Please show that this change is not breaking 4-byte unicode parsing.

I returned part of the code "4-byte unicode parsing" back. Thanks for your comment.
And I attach testsuite-utplsql result.
[testsuite-utplsql_result.txt](https://github.com/pljson/pljson/files/6997473/testsuite-utplsql_result.txt)